pkg/mountutil/mountutil.go

```diff
@@ -161,5 +158,150 @@ func ProcessFlagV(s string, volStore volumestore.VolumeStore, createDir bool) (*
 		}
 		res.Mount.Options = strutil.DedupeStrSlice(append(res.Mount.Options, unpriv...))
 	}
-	return &res, nil
+
+	return res, nil
+}
+
+func handleVolumeToMount(source string, volStore volumestore.VolumeStore, createDir bool) (volumeSpec, error) {
+	var res volumeSpec
+
+	res.Type = Bind
+	res.Source = source
+
+	// Handle named volumes
+	if isNamedVolume(source) {
+		return handleNamedVolumes(source, volStore)
+	}
+
+	// Handle named pipes
+	if isNamedPipe(source) {
+		res.Type = Npipe
```

Comment:
nit: this is not necessarily for this PR, but I feel like these types could have been `enum`s?

Reply:
Will create a separate PR for this.